docs/guides/soloproject/soloproject.md

```diff
@@ -40,10 +40,10 @@ best matches your current skill level - beginner (Tier 1),
 intermediate (Tier 2), or experienced (Tier 3).
 
 [Developer tiers](./topics/tier_developer.md)</br>
```

Comment:
Suggest changing this to `Developer (Web) tiers` to make it clear what the focus of this tier is.